src/DependencyInjection/FOSElasticaExtension.php

```diff
@@ -94,10 +94,8 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadClients($config['clients'], $container);
         $container->setAlias('fos_elastica.client', \sprintf('fos_elastica.client.%s', $config['default_client']));
-        $container->getAlias('fos_elastica.client')->setPublic(true);
-        $container->setAlias(ElasticaClient::class, new Alias('fos_elastica.client', false));
+        $container->setAlias(ElasticaClient::class, 'fos_elastica.client');
```

Comment:
It is still needed to mark it as public false I think for the moment ;)

Reply:
Hmm, you're right. Even in 5.1 the default value is public: true https://github.com/symfony/dependency-injection/blob/5.1/Alias.php#L25

I was under the impression all services were private by default since Symfony 3.4: https://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default

Reply:
services yes but not aliases ;)

Reply:
Well, the Symfony blog says:

> That's why in Symfony 3.4 services **and aliases** will be private by default.

Reply:
You're right but that's for aliases declared in configuration files not by using the container builder ;)

Reply:
I see.

Well that was confusing :) 

BTW is there any reason to use the Alias class?

```php
$container->setAlias(ElasticaClient::class, new Alias('fos_elastica.client', false));
```

vs

```php
$container->setAlias(ElasticaClient::class, 'fos_elastica.client')
    ->setPublic(false);
```

Reply:
```php
$container->setAlias(ElasticaClient::class, new Alias('fos_elastica.client', false));
```

That's what some bundle use, so that's fine and less confusing in this case ;)